otherlibs/stdune/src/ansi_color.mli

```diff
@@ -94,7 +94,11 @@ module Style : sig
   val escape_sequence : t list -> string
 end
 
-val make_printer : bool Lazy.t -> Format.formatter -> (Style.t list Pp.t -> unit) Staged.t
+val make_printer
+  :  line_break:bool
```

Comment:
How about we avoid changing the public API here for the tests? Not that the change itself is bad, but when we're changing the API just to accommodate 1 test out of hundreds, it's hard to justify this trade-off. If you can't use Dune_config, then so bet it. Feel free to use `Sys.getenv` directly. It's just for the tests anyway.

Reply:
In this particular case `make_printer` is only public for a test, the regular code uses `print` or `prerr`, thus I considered it not too invasive.

Anyway, I removed `Dune_config` and used `Sys.getenv` and reverted all these changes.